### ISSUE TITLE
Erroneous measurement from BatteryManager official interface

### ISSUE THREAD

**Issue by ste23droid** [community-code/amps#1]
Hi there,
I've tested your code on multiple devices and first of all thank you, works great!

However when I've tested your code on a Mi A1 device and on a P8 Lite 2017 device I found some discrepancy. Both those devices use the official Android interface provided by the Battery Manager to obtain the current measurement. 

On the Mi A1 the measurement is provided by the BatteryManager in **milliAmpere * 1000** and is **positive** when the phone is discarging, on the P8 Lite 2017 the measurement is **already provided in milliamperes** and is **negative** when the phone is discarging.

Basically here https://github.com/community-code/amps/blob/master/app/src/main/java/com/communitycode/amps/main/battery/OfficialBatteryMethod.java in same cases dividing by 1000 is not correct.

I attach some screenshots taken from my test.
Have a nice day![
![mia1](https://user-images.githubusercontent.com/11226457/45436050-279f8600-b6b2-11e8-9ef4-b4b353dc52f0.png)
![p8lite](https://user-images.githubusercontent.com/11226457/45436051-279f8600-b6b2-11e8-9fea-eae5202ddcd6.png)

](url)
